packages/runner-ct/src/specs/specs-list.jsx

```diff
@@ -0,0 +1,79 @@
+import { observer } from 'mobx-react'
+import React, { Component } from 'react'
+import specsStore from './specs-store'
+import SpecGroup from './spec-group'
+import StateContext from './spec-context'
+
+@observer
+class SpecsList extends Component {
+  render () {
+    const specGroups = specsStore.specs.length ? makeSpecHierarchy(specsStore.specs) : {}
+
+    return (
+      <div className="specs-list">
+        <header>Select tests to run...</header>
+        <StateContext.Provider value={{
```

Comment:
Let's not mix up context providers and mobx state. Unless we decided to completely rewrite test runner state lets use mobx as single source of trust

Reply:
It makes the code look more complicated that it should but I agree, let's use mobx as the single source of truth.